packages/create-instance/create-instance.js

```diff
@@ -57,6 +59,40 @@ export default function createInstance (
   addAttrs(vm, options.attrs)
   addListeners(vm, options.listeners)
 
+  if (options.scopedSlots) {
+    if (window.navigator.userAgent.match(/PhantomJS/i)) {
+      throwError('the scopedSlots option does not support strings in PhantomJS. Please use Puppeteer, or pass a component.')
+    }
+    const vueVersion = Number(`${Vue.version.split('.')[0]}.${Vue.version.split('.')[1]}`)
+    if (vueVersion >= 2.5) {
+      vm.$_vueTestUtils_scopedSlots = {}
+      vm.$_vueTestUtils_slotScopes = {}
+      const renderSlot = vm._renderProxy._t
+
+      vm._renderProxy._t = function (name, feedback, props, bindObject) {
+        const scopedSlotFn = vm.$_vueTestUtils_scopedSlots[name]
+        const slotScope = vm.$_vueTestUtils_slotScopes[name]
+        if (scopedSlotFn) {
+          props = { ...bindObject, ...props }
+          const proxy = {}
+          const helpers = ['_c', '_o', '_n', '_s', '_l', '_t', '_q', '_i', '_m', '_f', '_k', '_b', '_v', '_e', '_u', '_g']
+          helpers.forEach((key) => {
+            proxy[key] = vm._renderProxy[key]
+          })
+          proxy[slotScope] = props
```

Comment:
Does Vue throw an error if there are two conflicting slot scopes? If not, we should throw an error here

Reply:
It is impossible to set a word that starts with "_".

```
[Vue warn]: Property or method "_foo" is not defined on the instance but referenced during render. 
Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. 
See: https://vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.
```

Reply:
I meant if you mounted with this option:
```
{
scopedSlots: {
  foo: '<div />',
  foo: '<input />'
}
}
```

The second scopedSlot would overwrite the first scopedSlots in `_vueTestUtils_slotScopes`. Although thinking about it, this is probably fine.

Reply:
I will add a exception and document.

Reply:
It is not possible to register the same key.

```
{ foo: 1, foo: 2 }
// => { foo: 2 }
```

Reply:
🤦‍♂️ Of course ;p